recipes/fiftyone-brain/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fiftyone-brain" %}
+{% set version = "0.16.1" %}
```

Comment:
Latest version is 0.17.0, if you need this one specifically it is fine but if you want the latest, you can update it now.

Reply:
will I need to update the hashes and stuff?

will some bot send a PR updating it after this PR becomes a org repo?

Reply:
If you want to update it now, yes, you will need to update the version and the hash. In the feedstock the bot will send a PR for you.